Configuration/Objects.yaml

```diff
@@ -1,3 +1,33 @@
+Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\QueryInterface:
```

Comment:
It's a lots of Objects configuration, but the configuration is done from the Settings.yaml just by configuring the version of Elastic